src/test/java/org/jitsi/jibri/CallUrlInfoTest.kt

```diff
@@ -0,0 +1,44 @@
+package org.jitsi.jibri
+
+import org.testng.Assert.assertEquals
+import org.testng.Assert.assertFalse
+import org.testng.Assert.assertTrue
+import org.testng.annotations.Test
+
+class CallUrlInfoTest {
+    @Test
+    fun `test creation`() {
+        val info = CallUrlInfo("baseUrl", "callName")
+        assertEquals("baseUrl", info.baseUrl)
+        assertEquals("callName", info.callName)
+        assertEquals("baseUrl/callName", info.callUrl)
+    }
+
+    @Test
+    fun `test equals() and hashCode()`() {
```

Comment:
You weighed having two tests vs one test and decided it was worth it to keep the tests together?

Reply:
Also, is overriding "==" a kotlin thing or standard in some version of java?

Reply:
i did these as one because equals/hashcode are defined based on one another (hashcode must be equal if == returns true).  overriding == is in java as well.